### ISSUE TITLE
Bot Usage Guide on the Discord Server

### ISSUE THREAD

**Issue by umutucak** [Luxembourg-Open-Source-Club/BICS-BOT#50]
# Is your feature request related to a problem? Please describe.
Students are still confused on how to use the bot and enroll to classes. This must be dealt with ASAP. 

# Describe the solution you'd like
We need to make a text channel `#bot-usage-guide` where we ELI5 (explain-like-I'm-5) to the students how to use the commands.

**Comment by umutucak** [comment]
A great way to introduce people to the bot could be to use the **Welcome Screen** when people join the server. People can skip over text channels, but everyone MUST see the welcoming screen. A PSA could be put there highlighting the importance of using the bot + how to use it.

**Comment by PedroS235** [comment]
@umutucak This is done right?
